packages/cli/src/commands/license/clear.ts

```
@@ -12,10 +12,12 @@ export class ClearLicenseCommand extends BaseCommand {
 	async run() {
 		this.logger.info('Clearing license from database.');
 
-		// Invoke shutdown() to force any floating entitlements to be released
-		const license = Container.get(License);
-		await license.init();
-		await license.shutdown();
+		// Attempt to invoke shutdown() to force any floating entitlements to be released
+		try {
+			const license = Container.get(License);
+			await license.init();
+			await license.shutdown();
+		} catch {}
```

IMO we should at least log the error. Swallowing errors is not a good practice